### DISCUSSION TITLE
New here !

### DISCUSSION THREAD

**Discussion by kashh56** [camicroscope/caMicroscope#1049] [General]
Hi i am new to this community and open source in general can anyone tell me how to get started and contribute to this space ! Techincal skills that i have are knowledge about databases , machine learning , python, web development (little html css ) flask , django , fastapi... Open to learning new skills and apply it as well.. Currently on My deep learning journey..

**Comment by Rishi2003Das** [comment]
Hello everyone! I am *Rishi Das* from India. I am willing to participate as a *contributor in GSOC 2025* and while checking out the past projects I find your organization very interesting and relatable to my work. I am an undergrad engineering student studying in my 3rd year of graduation. I am very enthusiastic about working in the fields of AI/ML and I am also doing *research in this field of medical imaging. I mainly work on Computer Vision projects related to **Medical Imaging like Cancer detection and classification*. I have participated in a few competitions and hackathons too as an ML Engineer and did several challenges in Kaggle. It's my passion to build new model architectures and train them in large datasets. Hope you like my work and introduction. I am interested to contribute in your projects so, please mention what are the projects that you are aiming for in future to work for GSOC '25. So, that I can contribute to your project. 
Thanking you,
Rishi Das.
[LinkedIn](https://www.linkedin.com/in/rishi-das-522537250/)
[Kaggle](https://www.kaggle.com/rishi2003das)
[Email](rishikakalidas@gmail.com)
